docs/rackhd/install_os.rst

```diff
@@ -53,13 +53,26 @@ Check the workflow is active or inactive:
 
     curl <server>/api/1.1/nodes/<identifier>/workflows/active
 
+2.0 API Check the workflow is active or inactive:
```

Comment:
This is requesting what active workflows are against the node if I'm reading this correctly?